docs-2.0/8.service-tuning/improve-query-by-tag-index.md

```diff
@@ -1,26 +1,27 @@
-<!-->
-# 增加和删除标签
+# Add and delete tags
 
-在openCypher中，有增加标签（`SET label`）和移除标签（`REMOVE label`）的功能，可以用于加速查询或者标记过程。
+openCypher has the features of `SET label` and `REMOVE label` to speed up the process of querying or labeling.
```

Comment:
Use uppercase for the o in openCypher when it is at the beginning of the sentence. This is our preferred style.